themes/openy_themes/openy_carnation/src/js/openy_carnation.js

```diff
@@ -0,0 +1,460 @@
+      }
+    }
+  };
+
+  /**
+   * Ensure header alerts are after breadcrumbs in the DOM
+   */
+  Drupal.behaviors.moveHeaderAlerts = {
+    attach: function (context, settings) {
+      var headerAlerts = $('#block-openy-carnation-views-block-alerts-header-alerts', context);
+      var subHeaderFilters = $('.banner-cta', context);
+
+      if (headerAlerts.length && subHeaderFilters.length) {
+        headerAlerts.once('openy-breadcrumbs').insertAfter(subHeaderFilters);
+      }
+    }
+  };
+
+  /**
+   * Show/hide desktop search block.
+   */
+  Drupal.behaviors.openySearchToggle = {
+    attach: function (context, settings) {
+      var searchBtn = $('.site-search button');
+      var mainMenuLinks = $('.page-head__main-menu .nav-level-1 li:not(:eq(0))').find('a, button');
+      var searchClose = $('.page-head__search-close');
+
+      searchBtn.once('openy-search-toggle-hide').on('click', function () {
+        mainMenuLinks.removeClass('show').addClass('fade');
+      });
+
+      searchClose.once('openy-search-toggle-show').on('click', function () {
+        mainMenuLinks.addClass('show');
+      });
+    }
+  };
+
+  /**
+   * Add class to header when mobile menu is opened.
+   */
+  Drupal.behaviors.openyMobileMenu = {
+    attach: function (context, settings) {
+      var sidebar = $('#sidebar');
+      var $target = $('.top-navs');
+
+      sidebar.on('show.bs.collapse', function () {
+        $target.addClass('menu-in');
+      });
+
+      sidebar.on('hide.bs.collapse', function () {
+        $target.removeClass('menu-in');
+      });
+    }
+  };
+
+  /**
+   * Dropdown menu height.
+   */
+  Drupal.behaviors.openyDropdownMenu = {
+    attach: function (context, settings) {
+      $('.nav-desktop .nav-level-2').each(function (index, element) {
+        var item = $(element);
+        var offset = item.offset();
+        var bottom = offset.top + item.height() - $(window).scrollTop();
+        var maxHeight = $(window).height() - bottom - 15;
+        item.find('.dropdown-menu').eq(0).css('max-height', maxHeight);
+      });
+    }
+  };
+
+  /**
+   * BS4 data-spy: affix replacement
+   */
+  Drupal.behaviors.openyHeaderAffix = {
+    attach: function (context, settings) {
+      $(window).on('scroll', function(event) {
+        var scrollValue = $(window).scrollTop();
+        if (scrollValue === settings.scrollTopPx || scrollValue > 1) {
+          $('.top-navs').addClass('affix');
+        } else if (scrollValue === settings.scrollTopPx || scrollValue < 1) {
+          $('.top-navs').removeClass('affix');
+        }
+      });
+    }
+  };
+
+  /**
+   * Branch Locations (Download PDF)
+   */
+  Drupal.behaviors.openyPdfDownload = {
+    attach: function (context, settings) {
+      var pdfBtnContainer = $('.groupex-pdf-link-container');
+      if (pdfBtnContainer.length) {
+        pdfBtnContainer.find('a').html('PDF <i class="fas fa-download"></i>');
+        pdfBtnContainer.insertAfter('.groupex-form-full .form-submit');
```

Comment:
suggest adding a class to the link and insert icon with CSS

see GroupexHelper::getPdfLink()

Reply:
This will change it for all themes @ygerasimov, don't think I have approval for a global change here.

Reply:
Just to confirm @podarok, you'd like an inline JS declaration in the twig template for this?

Reply:
@aaronjchristian i see a code insertAfter
If you add some class like ```carnation-theme``` to html or body - this code will run only there. 
That was an idea